src/lib/wallet.ts

```diff
@@ -20,6 +20,8 @@ import { makeInteractiveSigner } from './keyManagement.js';
 import { boardSlottingMarshaller, networkConfigUrl, RpcUtils } from './rpc';
 import { makeRpcUtils } from './rpc.js';
 
+export type RelativeTime = { timerBrand: Brand; relValue: bigint };
```

Comment:
please import from @agoric/time: https://github.com/Agoric/agoric-sdk/blob/f17614a7376d335562c88845a4cc457da0805b7c/packages/time/src/types.d.ts#L61-L65

Reply:
The import doesn't seem to be playing nice